core/src/main/scala/filodb.core/metadata/Column.scala

```diff
@@ -31,12 +34,17 @@ trait Column {
  */
 case class DataColumn(id: Int,
                       name: String,
-                      columnType: Column.ColumnType) extends Column {
+                      columnType: Column.ColumnType,
+                      params: Config = ConfigFactory.empty) extends Column {
+  import collection.JavaConverters._
```

Comment:
We should add some validation for histogram columns to ensure presence of the metadata.

Reply:
ok I'll add a validation API.

Reply:
@vishramachandran added, have a look.